content/guides/contribute/design.mdx

```diff
@@ -1,70 +1,56 @@
+Finally, consider where a pattern will live in the context of a system. A pattern designed for a specific context is not likely to be considered a reusable component.
 
 ## Design mockups and prototypes
 
-@todo Coming soon!
-
-<!-- Get started in Figma by duplicating the **Primer pattern template** file. You can find it in **Design Infrastructure > Contributions**. Follow the instructions in the file on how to name, and use the template, so that it accurately indicates the stage of the design work.
-
-You can also use an alternative prototyping tool like CodePen, Framer, or anything else that you're comfortable using — just make sure the prototypes are linked to from the pattern tracking issue. -->
-
 ➡️ [Using Figma at GitHub](/guides/figma)
 
-<!-- @todo Naming, API: design and eng alignment -->
-
 ## Share early, and often
 
-The work that you're doing shouldn't be a surprise that you only reveal when you're done.
-
-**Systems designers** can provide extra context and considerations that are helpful to create a solid design.
-
-**Product designers** can consider the designs in the context of their own product area and whether their specific use cases could benefit from your proposal.
+There are many opportunities to get feedback from Primer maintaners. 
 
-**Engineers** can consider how something may be implemented, what is realistic within the existing constraints, and what is unfeasible.
+**Systems designers** provide extra context and considerations for designing with system-wide considerations
+**Accessibility designers** provide context and considerations for accessible and inclusive design
+**Product designers** consider the designs in the context of their own product area and whether their specific use cases could benefit from your work
+**Design Engineers** can consider how something may be implemented, what is realistic within the existing constraints, and what is unfeasible.
 
 There are different ways you can ask for and get feedback:
 
-- Async via [discussions](https://github.com/github/primer/discussions/categories/feedback) (GitHub staff only): Record a short video and/or links to a Figma prototype (or any other kind of prototype, like sketches or code), providing context and specifying what kind of feedback you're looking for
-- Primer open office hours: Attend office hours on Thursdays (1.30pm PT). The Zoom link is posted right before in #primer. Be prepared to give context and ask what kind of feedback you're looking for succinctly. Ideally, give a heads-up in #primer beforehand so we can be sure a designer will be present in the call
-- Ad-hoc in real-time: In #primer, ask for a short critique session with a systems designer, to get feedback on a specific issue
+- (GitHub staff only) [Open an issue requesting a Primer Design Review](https://github.com/github/primer/#need-a-primer-pattern-or-octicon-)
+- (GitHub staff only) Attend Primer or Primer Accessibility open office hours
```

Comment:
Should we mention the Primer patterns working session?

Reply:
The open an issue takes you to the inbox, which will land you in a primer patterns session if the issue is complex enough. this feels more evergreen.